check_tool_connection_frequency: 2.0
tools_params:
    id_list: [11,12,13,14,31]
    motor_type_list: ["xl320","xl320","xl320","xl320","xl320"]
